badger/cmd/info.go

```diff
@@ -515,6 +529,19 @@ func pluralFiles(count int) string {
 	return "files"
 }
 
+// When the checkSum Algorithm is invalid, func strToChecksumAlgorithm will return the default checkSum Algorithm
```

Comment:
What is benefit of returning the default checksum when we are returning the error along with it? I think we should return default checksum with nil error